health.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="description" content="This is a boilerplate for building websites." />
```

Comment:
Update your name and content attribute values to be descriptive of yoru website.